apache/zeppelin
# [ZEPPELIN-6112] Fix running documentation website server

### What is this PR for?
The current command for running the website is not functioning correctly.

Fixing this issue will enable developers to verify that the changes made to the documentation files are properly reflected on the website.


### What type of PR is it?
Bug Fix

### Todos

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/ZEPPELIN/6112
* Put link here, and add [ZEPPELIN-*Jira number*] in PR title, eg. [ZEPPELIN-533]

### How should this be tested?

Run the website server using the following command and check if the site is served at http://localhost:4000:
```bash
docker run --rm -it \
       -v $PWD:/docs \
       -w /docs \
       -p '4000:4000' \
       ruby:3.3.5 \
       bash -c "bundle install && bundle exec jekyll serve --watch --host 0.0.0.0"
```

### Questions:
* Does the license files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
